include/triton/core/tritonserver.h

```diff
@@ -546,7 +548,8 @@ typedef enum tritonserver_traceactivity_enum {
   TRITONSERVER_TRACE_COMPUTE_INPUT_END = 3,
   TRITONSERVER_TRACE_COMPUTE_OUTPUT_START = 4,
   TRITONSERVER_TRACE_COMPUTE_END = 5,
-  TRITONSERVER_TRACE_REQUEST_END = 6
+  TRITONSERVER_TRACE_REQUEST_END = 6,
+  TRITONSERVER_TRACE_TENSOR = 7
```

Comment:
We will want to collect tensor trace in a couple of places, to the enum should be specific. For starters perhaps TRITONSERVER_TRACE_TENSOR_INPUT and TRITONSERVER_TRACE_TENSOR_OUTPUT